combainer/client.go

```diff
@@ -199,32 +174,25 @@ func (cl *Client) Dispatch(parsingConfigName string, uniqueID string, shouldWait
 		"session": uniqueID,
 		"config":  parsingConfigName}
 
-	var deadline, startTime time.Time
+	var startTime time.Time
```

Comment:
I would like to do below simple `startTime :=`
and define wg just before `for`.